### PR TITLE
PR: Update link to demo app from Heroku to Fly.io issue #68

### PR DESCRIPTION
**Pull request by nelsonic**
Deployed to fly.io via https://github.com/dwyl/elixir-auth-github-demo/issues/22 
This PR just updates the link and exports the `/config` for testing convenience. #68

### CODE REVIEW

**Comment by nelsonic** [comment]
https://github.com/dwyl/elixir-auth-github/actions/runs/3663119720/jobs/6192690508
image


```sh
erlef/setup-elixir@88597[1](https://github.com/dwyl/elixir-auth-github/actions/runs/3663119720/jobs/6192690508#step:3:1)a[7](https://github.com/dwyl/elixir-auth-github/actions/runs/3663119720/jobs/6192690508#step:3:8)2ed1f9240973bd92ab57af8c1aa68f24
Installing OTP OTP-24.0.2 - built on ubuntu-18.04
Installing Elixir v1.12.3
/home/runner/work/_temp/.setup-elixir/elixir/bin/mix local.rebar --force
/home/runner/work/_temp/.setup-elixir/otp/erts-12.0.2/bin/beam.smp: error while loading shared libraries: libtinfo.so.5: cannot open shared object file: No such file or directory
Error: The process '/home/runner/work/_temp/.setup-elixir/elixir/bin/mix' failed with exit code 127
```

hmm ... 🙄 
gonna update the versions of Elixir & OTP and hopefully this resolves itself. 🤞

**Comment by nelsonic** [comment]
Dependent on: https://github.com/dwyl/elixir-auth-github-demo/pull/28

**Comment by nelsonic** [comment]
```elixir
defmodule ElixirAuthGithub.HTTPoisonMock do
  @doc """
  `get/3` stubs the HTTPoison get! function when parameters match test vars.
  """
  @valid_body %{
    access_token: "12345",
    login: "test_user",
    name: "Testy McTestface",
    email: "test#{:rand.uniform(1_000_000)}@gmail.com",
    avatar_url: "https://avatars3.githubusercontent.com/u/10835816",
    id: "19"
  }
```

##  `email: "test#{:rand.uniform(1_000_000)}@gmail.com",`

File: README.md [hunk 1]
```diff
@@ -56,9 +56,9 @@ by following these 5 simple steps:
 
 > If you get stuck setting up your App,
 > checkout our working demo:
-> https://github.com/dwyl/elixir-auth-github-demo <br />
-> The demo is deployed on Heroku:
-> https://elixir-auth-github-demo.herokuapp.com/
+> [dwyl/elixir-auth-github-demo](https://github.com/dwyl/elixir-auth-github-demo) <br />
+> The demo is deployed on Fly.io:
+> https://elixir-auth-github-demo.fly.dev
```
Comment (nelsonic) [inline README.md:61]: Working as expected: https://elixir-auth-github-demo.fly.dev
image

**Review by SimonLab — APPROVED** [review]
LGTM

**Pull request merged**